mon-pix/app/services/base-url.js

```diff
@@ -0,0 +1,131 @@
+// This file is a copy of an original file from mon-pix.
+// If you need a change, modify the original file and
+// propagate the changes in the copies in all the fronts.
+
+import Service, { service } from '@ember/service';
+import ENV from 'mon-pix/config/environment';
+
+const { DEFAULT_LOCALE } = ENV.APP;
+
+// Don't use directly this service, use a url service which extends it
+export default class BaseUrlService extends Service {
+  @service currentDomain;
+  @service locale;
+
+  pixAppUrlWithoutExtension = ENV.APP.PIX_APP_URL_WITHOUT_EXTENSION;
+
+  get homeUrl() {
+    const currentLanguage = this.locale.currentLocale;
+    return `${ENV.rootURL}?lang=${currentLanguage}`;
```

Comment:
Cette suggestion nous rapproche de la cible même si elle est imparfaite car il n’y a pas une correspondance parfaite entre la locale et le language, mais le code original proposé a le même défaut de non-correspondance parfaite entre la locale et le language.
```suggestion
    const currentLocale = this.locale.currentLocale;
    return `${ENV.rootURL}?lang=${currentLocale}`;
```